client/Windows/wf_floatbar.c

```diff
@@ -25,6 +25,7 @@
 #include "wf_client.h"
 #include "wf_floatbar.h"
 #include "wf_gdi.h"
+#pragma comment( lib, "Msimg32.lib" )
```

Comment:
Given that the default value that CMake puts in the VS project files for WINVER and _WIN32_WINNT is 0x0501 (WIN32_WINNT_WINXP), it shouldn't even compile if those API's weren't supported.